Description: Step back to Debian architectural baseline
 .
 libcryptx-perl is build with the compiler flags -sse4.1 and -maes.
 SSE4.1 is supported by Intel CPUs, but not by all AMD CPUs. AMD CPUs
 support SSE4a, which is not supported by Intel CPUs. The Debian
 architectural baseline therefore states:
 .
 amd64         x86_64 with no optional extensions (psABI baseline). The
               core specification includes MMX, SSE and SSE2 so these
               are OK, but SSE3 and up are not guaranteed.
 
               See psABI (Table 3.1: Micro-Architecture Levels) for
               amd64 ABI level.
 .
 Remove the SSE4.1 and AES compiler flags.
Bug-Debian: https://bugs.debian.org/1056399
Forwarded: not-needed
Author: Benedikt Spranger <b.spranger@linutronix.de>
Reviewed-by: gregor herrmann <gregoa@debian.org>
Last-Update: 2023-11-24

--- a/Makefile.PL
+++ b/Makefile.PL
@@ -38,7 +38,6 @@
     $ver1 ||= $1 if $Config{gccversion} =~ /^([0-9]+)\./; # gccversion='10.2.0'
     $ver1 ||= $1 if $Config{gccversion} =~ /LLVM ([0-9]+)\./i; # gccversion='Apple LLVM 14.0.0 (clang-1400.0.29.202)'
     $ver1 ||= $1 if $Config{gccversion} =~ /Clang ([0-9]+)\./i; # gccversion='FreeBSD Clang 13.0.0 (git@github.com:llvm/llvm-project.git llvmorg-13.0.0-0-gd7b669b3a303)' or 'OpenBSD Clang 13.0.0'
-    $mycflags .= " -msse4.1 -maes" if $ver1 > 4; # supported since gcc-4.4
   }
 
   #FIX: this is particularly useful for Debian https://github.com/DCIT/perl-CryptX/pull/39
